packages/messages/common.d.ts

```diff
@@ -61,6 +61,12 @@ export type ChargingSchedule = {
   startSchedule?: Timestamp;
 };
 
+export type KeyValue = {
+  key: string;
+  readonly: boolean;
```

Comment:
maybe we can rename this as `isReadonly` to prevent problems with the reserved word

Reply:
but I don't know about mapping the OCPP message to this

Reply:
that's the original ocpp message typing